src/devices/Max31856/README.md

```diff
@@ -1,13 +1,42 @@
 ﻿# Max31856 - cold-junction compensated thermocouple to digital converter
 
-## Summary
 The Max31856 device is a SPI interface cold-junction compensated thermocouple to digital converter.
 
-## Sensor Image
 ![Illustration of wiring from a Raspberry Pi device](device.jpg)
 
 **Note:** _ThermocoupleType.K is configured for a K type thermocouple if you want to use a B,E,J,K,N,R,S, or T simply change the K to the thermocouple type of your choosing._
 
-## References 
+## Documentation 
 
 **Max31856** [datasheet](https://datasheets.maximintegrated.com/en/ds/Max31856.pdf)
+
+## Usage
+The Max31856.samples file contains a sample usage of the device. Note that this reads two temperatures. One is a connected thermocouple reading which can be read using the  ```TryGetTemperature``` command and the other is the temperature of the device itself which can be read using the ```GetColdJunctionTemperature``` command. The Cold Junction Temperature is used internally to increase the accuracy of the thermocouple but can also be read if you find a use for it.
```

Comment:
Marktdown tips:

- all titles except the first level one, must have an empty line and an empty line after.
- all code blocks need to be separated with an empty line before and after
- al enums need to be grouped without extra enter between them
- for code blocks, use only "csharp" and not the c# version, all lowercase